### PR TITLE
feat:send message with enter button and to show active or offline

### PR DESCRIPTION
**Pull request by Nue12**
### What I have done?

- add functionality to send a message with the "Enter" button on the keyboard
- to show active or offline for the users (codes in "index.ts")

### CODE REVIEW

File: frontend/src/pages/home/index.js [hunk 1]
```diff
@@ -84,6 +106,128 @@ function closeUsersMenu() {
...
+    socket.emit("active", userName);
+    console.log(socket.connected);
+  });
+
+  socket.on("disconnect", () => {
+    socket.emit("offline", userName);
+  });
+
+  inputMsg.addEventListener("keyup", (e) => {
+    if (e.key === "Enter") {
```
Comment (zwelhtetyan) [inline frontend/src/pages/home/index.js:125]: need to check:  if (!inputMsg.value)

File: frontend/src/pages/home/index.js [hunk 2]
```diff
@@ -84,6 +106,128 @@ function closeUsersMenu() {
...
+        message: inputMsg.value,
+        userName: userName,
+      };
+      socket.emit("chat", data);
+      inputMsg.value = "";
+    }
+  });
+
+  sendBtn.addEventListener("click", () => {
+    const data = {
```
Comment (zwelhtetyan) [inline frontend/src/pages/home/index.js:136]: need to check if value is empty

File: frontend/src/pages/home/index.html [hunk 3]
```diff
@@ -53,18 +58,23 @@
...
               <div class="flex space-x-2.5 items-baseline">
                 <p class="font-medium">Zwel</p>
                 <span>Today at 1:52 PM</span>
               </div>
-              <!-- message -->
               <p class="ml-5">br loat nay kya ll</p>
-            </div>
+            </div> -->
 
             <!-- other messages -->
```
Comment (zwelhtetyan) [inline frontend/src/pages/home/index.html:80]: latest message should be scroll into view

**Comment by zwelhtetyan** [comment]
I have done adding auto scroll and check for empty input.  Plese check this [PR](https://github.com/zwelhtetyan/chat-app/pull/11).

File: frontend/src/pages/home/index.js (hunk 2 above)
Comment (Nue12) [reply frontend/src/pages/home/index.js:136]: Ok sir

**Comment by Nue12** [comment]
I fixed to check the values of input messages

**Review by zwelhtetyan — APPROVED** [review]

**Pull request closed without merging**